deploy/crds/core_v1_storagecluster_crd.yaml

```diff
@@ -1361,6 +1372,52 @@ spec:
                     type: integer
                     format: int64
                     description: The number of storage nodes per zone in the cluster.
+              desiredImages:
+                type: object
+                description: Represents all the desired images of various components.
+                properties:
+                  stork:
+                    type: string
+                    description: Desired image for stork.
+                  userInterface:
```

Comment:
This was used for Portworx Lighthouse - our original GUI dashboard. Since PX-Central, we have stopped development of it and not recommending it to customers.